cmd/sendevent/main.go

```diff
@@ -41,7 +41,7 @@ var (
 func init() {
 	flag.StringVar(&eventID, "event-id", "", "Event ID to use. Defaults to a generated UUID")
 	flag.StringVar(&eventType, "event-type", "google.events.action.demo", "The Event Type to use.")
-	flag.StringVar(&source, "source", "", "Source URI to use. Defaults to the current machine's hostname")
+	flag.StringVar(&source, "source", "sendevent", "Source URI to use. Defaults to the current machine's hostname")
```

Comment:
is the default really right? Seems like the default is "sendevent" and not the current machine's hostname?

Reply:
I just needed a value to pass ce validation.